cms/templates/js/mock/mock-group-configuration-page.underscore

```diff
@@ -5,22 +5,21 @@
                 <small class="subtitle">${_("Settings")}</small>
                 <span class="sr">&gt; </span>${_("Group Configurations")}
             </h1>
-            <nav class="nav-actions">
-                <h3 class="sr">${_("Page Actions")}</h3>
-                <ul>
-                    <li class="nav-item">
-                        <a href="#" class="button new-button"><i class="icon-plus"></i> ${_("New Group Configuration")}</a>
-                    </li>
-                </ul>
-            </nav>
         </header>
     </div>
 
     <div class="wrapper-content wrapper">
         <section class="content">
             <article class="content-primary" role="main">
-                <div class="ui-loading">
-                    <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading...")}</span></p>
+                <div class="wrapper-groups cohort-groups">
+                    <div class="ui-loading">
+                        <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading...")}</span></p>
```

Comment:
Also, why not leverage the ``icon-spin`` class on the icon rather than our spin animation on the parent element?